packages/apps/transfer/src/services/modules/list-module.ts

```diff
@@ -0,0 +1,25 @@
+import { PactCommand } from '@kadena/client';
+import { createExp } from '@kadena/pactjs';
+
+//List modules PACT code example
+
+const NETWORK_ID: any = 'testnet04';
+const chainId: string = '1';
+const gasLimit: number = 6000;
+const gasPrice: number = 0.001;
+const ttl: number = 600;
+export const API_HOST: string = `https://api.testnet.chainweb.com/chainweb/0.0/${NETWORK_ID}/chain/${chainId}/pact`;
```

Comment:
Idem. Env vars?

Reply:
i will implement a whole config service for this one soon